python_scripts/libaerie/products/product_parser.py

```diff
@@ -328,13 +544,31 @@ def check_event(cls, event_hash: dict) -> bool:
 
   @classmethod
   def cast_event(cls, event_hash: dict) -> str:
```

Comment:
Pardon my ignorance - what is an event hash? I'm a bit surprised to see something called a hash with type dict - typically hashes are integers or strings

Reply:
All good, I've gotten so used to calling them dictionaries I didn't think of the word hashmap. Maybe "dict" is the most pythonic abbreviation